x/fee_grant/internal/types/codec.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/fee_grant/exported"
+)
+
+// RegisterCodec registers the account types and interface
+func RegisterCodec(cdc *codec.Codec) {
+	cdc.RegisterInterface((*exported.FeeAllowance)(nil), nil)
+	cdc.RegisterConcrete(&BasicFeeAllowance{}, "feegrant/BasicFeeAllowance", nil)
+}
+
+// ModuleCdc generic sealed codec to be used throughout module
+var ModuleCdc *codec.Codec
+
+func init() {
+	cdc := codec.New()
+	RegisterCodec(cdc)
+	codec.RegisterCrypto(cdc)
+	ModuleCdc = cdc.Seal()
```

Comment:
question: do we want to seal the codec?

Reply:
Cuz that is what is done in all standard modules that I modeled upon. eg:
https://github.com/cosmos/cosmos-sdk/blob/2222d0eb811066494da0758e416d200c9583639a/x/bank/internal/types/codec.go#L16-L21